catalog/resource_metastore_test.go

```diff
@@ -123,6 +123,7 @@ func TestCreateMetastore_DeltaSharing(t *testing.T) {
 					DeltaSharingScope: "INTERNAL_AND_EXTERNAL",
 					DeltaSharingRecipientTokenLifetimeInSeconds: 0,
 					DeltaSharingOrganizationName:                "acme",
+					ForceSendFields:                             []string{"DeltaSharingRecipientTokenLifetimeInSeconds"},
```

Comment:
This test compares the marshalled send data. Since now we are sending DeltaSharingRecipientTokenLifetimeInSeconds, we need to add it to FSF to the expected request.

Reply:
That's awesome. So before, the test wasn't actually checking if this attribute was in the request (as we can't distinguish between absent and set to zero value in Go), but now we can assert that it was sent and set to the zero value.

Reply:
Correct. Before it was just checking that it was "not non-zero".